src/actions/feedback-form.js

```diff
@@ -26,12 +26,12 @@ export function toggleFeedbackValid () {
  * @param {Object} data
  * @returns {Function}
  */
-export function submitFeedback (theUrl, data) {
+export function submitFeedback (data) {
   return (dispatch, getState) => {
-    const theStore = getState();
-    if (typeof theStore.KmtHeaderNs.helpers.doRequest === 'function') {
-      const options = {method: 'POST', body: JSON.stringify(data), headers: { 'Content-Type': 'application/json' }, 'mode': 'no-cors'};
-      theStore.KmtHeaderNs.helpers.doRequest(theUrl, options).then(() => {
+    // const theStore = getState();
+    // if (typeof theStore.KmtHeaderNs.helpers.doRequest === 'function') {
+      // const options = {method: 'POST', body: JSON.stringify(data), headers: { 'Content-Type': 'application/json' }, 'mode': 'no-cors'};
```

Comment:
Should this all be commented out? Can it be deleted?

Reply:
✅